src/MessageOutbox/RelayMessagesThroughDispatcher.php

```diff
@@ -0,0 +1,84 @@
+<?php
+declare(strict_types=1);
+
+namespace EventSauce\MessageOutbox;
+
+use EventSauce\BackOff\BackOffStrategy;
+use EventSauce\BackOff\ExponentialBackOffStrategy;
+use EventSauce\EventSourcing\Message;
+use EventSauce\EventSourcing\MessageDispatcher;
+use Throwable;
+use Traversable;
+
+use function count;
+
+final class RelayMessagesThroughDispatcher
```

Comment:
@frankdejonge not sure if you want to have it `final` or not.
I can drop the modifier if you prefer.

Reply:
Final is good